### DISCUSSION TITLE
add import skin from osu!stable button & be able to change skin in setup wizard

### DISCUSSION THREAD

**Discussion by anispwyn** [ppy/osu#18280] [Ideas] [answered]
thats is a common feature to have in setup wizard

*i'm sorry for making two discussion in a short time, i feels bad because it looks like i'm spamming discussion*

**Comment by peppy** [comment, marked as answer]
This actually reveals a bit of a regression, where if you run through first run setup song select will no longer prompt to do a full import-from-stable (because there are non-zero beatmaps).

**Reply by frenzibyte** [reply]
https://github.com/ppy/osu/pull/18313 should handle this pretty well (along with actually addressing this thread entirely), removing the one in song select entirely.
